src/west/configuration.py

```diff
@@ -616,7 +616,7 @@ def _location(cfg: ConfigFile, topdir: Optional[PathType] = None,
             return env['WEST_CONFIG_LOCAL']
 
         if topdir:
-            return os.fspath(Path(topdir) / '.west' / 'config')
+            return os.fspath(Path(topdir) / WEST_DIR / 'config')
```

Comment:
Thanks, applied your suggestion here.

Reply:
Had to revert here, it's possible that the path does not exist yet. So it would require more changes.

I'll create a follow-up PR for v1.4 or v2.

Reply:
> Had to revert here, it's possible that the path does not exist yet. So it would require more changes.

As usual, initialization/"bootstrapping" is at the top of complex situations. Always underestimated :-)